staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go

```diff
@@ -374,13 +368,13 @@ func (o *PortForwardOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, arg
 }
 
 // Validate validates all the required options for port-forward cmd.
-func (o PortForwardOptions) Validate() error {
+func (o PortForwardOptions) Validate(cmd *cobra.Command) error {
```

Comment:
PortForwardOptions can be used independent of the command... making consumers pass in a dummy cmd object doesn't seem great. The existing invocation seems better to me.

Reply:
Thanks for your repay. 
Since the validate message is clear enough to user, we can conside removing the cmdutil.UsageErrorf() call.